[AirPlay 2] `libplist` 2.2.0 installed via Arch Linux's `pacman` is not detected by configure script

I have installed `libplist` via `pacman`:

```
$ sudo pacman -Qi libplist
Name            : libplist
Version         : 2.2.0-3
Description     : A library to handle Apple Property List format whereas it's binary or XML
Architecture    : x86_64
URL             : https://libimobiledevice.org/
Licenses        : GPL2  LGPL2.1
Groups          : None
Provides        : None
Depends On      : glib2
Optional Deps   : None
Required By     : libusbmuxd
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 417.68 KiB
Packager        : Evangelos Foutras <foutrelis@archlinux.org>
Build Date      : Mi 11 Nov 2020 16:06:11 CET
Install Date    : Mi 24 Nov 2021 15:54:08 CET
Install Reason  : Installed as a dependency for another package
Install Script  : No
Validated By    : Signature
```

This is where `libplist` was installed:

```
$ pacman -Ql libplist     
libplist /usr/
libplist /usr/bin/
libplist /usr/bin/plistutil
libplist /usr/include/
libplist /usr/include/plist/
libplist /usr/include/plist/Array.h
libplist /usr/include/plist/Boolean.h
libplist /usr/include/plist/Data.h
libplist /usr/include/plist/Date.h
libplist /usr/include/plist/Dictionary.h
libplist /usr/include/plist/Integer.h
libplist /usr/include/plist/Key.h
libplist /usr/include/plist/Node.h
libplist /usr/include/plist/Real.h
libplist /usr/include/plist/String.h
libplist /usr/include/plist/Structure.h
libplist /usr/include/plist/Uid.h
libplist /usr/include/plist/cython/
libplist /usr/include/plist/cython/plist.pxd
libplist /usr/include/plist/plist++.h
libplist /usr/include/plist/plist.h
libplist /usr/lib/
libplist /usr/lib/libplist++-2.0.so
libplist /usr/lib/libplist++-2.0.so.3
libplist /usr/lib/libplist++-2.0.so.3.3.0
libplist /usr/lib/libplist-2.0.so
libplist /usr/lib/libplist-2.0.so.3
libplist /usr/lib/libplist-2.0.so.3.3.0
libplist /usr/lib/pkgconfig/
libplist /usr/lib/pkgconfig/libplist++-2.0.pc
libplist /usr/lib/pkgconfig/libplist-2.0.pc
libplist /usr/lib/python3.9/
libplist /usr/lib/python3.9/site-packages/
libplist /usr/lib/python3.9/site-packages/plist.so
libplist /usr/share/
libplist /usr/share/man/
libplist /usr/share/man/man1/
libplist /usr/share/man/man1/plistutil.1.gz
```

However, the package doesn't seem to be detected by the `configure` script:

```
$ autoreconf -fi
...
$ ./configure --prefix=/usr --sysconfdir=/etc --with-alsa --with-pa --with-avahi --with-ssl=openssl --with-metadata --with-soxr --with-systemd --with-airplay-2
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for ranlib... ranlib
checking for git... git
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether the compiler supports GNU C++... yes
checking whether g++ accepts -g... yes
checking for g++ option to enable C++11 features... none needed
checking dependency style of g++... gcc3
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for clock_gettime in -lrt... yes
checking for pthread_create in -lpthread... yes
checking for exp in -lm... yes
checking for popt... yes
checking for libconfig... yes
checking for SSL... yes
checking for soxr... yes
checking for avahi_client_new in -lavahi-client... yes
checking for avahi_strerror in -lavahi-common... yes
checking for alsa... yes
checking for PULSEAUDIO... yes
>>Include libraries required for AirPlay 2
checking for libplist... no
configure: error: AirPlay 2 support requires libplist 2.0.0 or later -- libplist-dev suggested!
```

Any ideas?

--- a/configure.ac
+++ b/configure.ac
@@ -403,12 +403,11 @@ AC_ARG_WITH(airplay-2, [AS_HELP_STRING([--with-airplay-2],[Build for AirPlay 2])
 if test "x$with_airplay_2" = "xyes" ; then
   AC_DEFINE([CONFIG_AIRPLAY_2], 1, [Build for AirPlay 2])
   AC_MSG_RESULT(>>Include libraries required for AirPlay 2)
-  if test "x${with_os}" = xlinux  ; then
-      PKG_CHECK_MODULES([libplist], [libplist >= 2.0.0],[CFLAGS="${libplist_CFLAGS} ${CFLAGS}" LIBS="${libplist_LIBS} ${LIBS}"],[AC_MSG_ERROR(AirPlay 2 support requires libplist 2.0.0 or later -- libplist-dev suggested!)])
-  fi
-  if test "x${with_os}" = xfreebsd  ; then
-      PKG_CHECK_MODULES([libplist], [libplist-2.0 >= 2.0.0],[CFLAGS="${libplist_CFLAGS} ${CFLAGS}" LIBS="${libplist_LIBS} ${LIBS}"],[AC_MSG_ERROR(AirPlay 2 support requires libplist 2.0.0 or later -- search for pkg libplist-2.2.0 or later!)])
-  fi
+  PKG_CHECK_MODULES([libplist], [libplist >= 2.0.0],[CFLAGS="${libplist_CFLAGS} ${CFLAGS}" LIBS="${libplist_LIBS} ${LIBS}"],[
+    PKG_CHECK_MODULES([libplist], [libplist-2.0 >= 2.0.0],[CFLAGS="${libplist_CFLAGS} ${CFLAGS}" LIBS="${libplist_LIBS} ${LIBS}"],[
+      AC_MSG_ERROR(AirPlay 2 support requires libplist 2.0.0 or later -- search for pkg libplist-dev on Debian or libplist-2.2.0 or later on FreeBSD!)
+    ])
+  ])
   PKG_CHECK_MODULES([libsodium], [libsodium],[CFLAGS="${libsodium_CFLAGS} ${CFLAGS}" LIBS="${libsodium_LIBS} ${LIBS}"],[AC_MSG_ERROR(AirPlay 2 support requires libsodium -- libsodium-dev suggested)])
   AC_CHECK_LIB([gcrypt], [gcry_control], [], [AC_MSG_ERROR([Airplay 2 support requires libgcrypt -- libgcrypt-dev suggested])])
 #  PKG_CHECK_MODULES([libgcrypt], [libgcrypt],[CFLAGS="${libgcrypt_CFLAGS} ${CFLAGS}" LIBS="${libgcrypt_LIBS} ${LIBS}"],[AC_MSG_ERROR(AirPlay 2 support requires libgcrypt -- libgcrypt-dev suggested)])

